src/Debugging/Microsoft.DiaSymReader.PortablePdb.Tests/project.json

```diff
@@ -1,8 +1,4 @@
 {
-  "supports": {
```

Comment:
The package you list to get the platform definitions isn't something contained in the meta packages IIRC, so you might want to reference them regardless.